### ISSUE TITLE
"Favourites Mix" gets rid of window controls and back button

### ISSUE THREAD

**Issue by slunat** [Apple-Music-Electron/Apple-Music-Electron#181]
When clicking into Favourites Mix, the top window bar disappears and so does the back button. The bar goes transparent and you can interact with the objects underneath it, so its like the bar no longer exists. This doesn't seem to happen when clicking into any other playlists or albums, just the Favourites Mix on the Listen Now page. If I press the side button on my mouse to go back a page, the window control buttons and the back button will return. It just seems to be the Favourites Mix that breaks.

Running on Windows 10 with the window controls set to MacOS style on the right.

**Comment by cryptofyre** [comment]
Interesting. I'll look into it.

**Comment by greenfrogs** [comment]
I am not sure if this is related but the preferences button also seems to vanish in this mode. 

![image](https://user-images.githubusercontent.com/5961364/129741601-7936612b-c0bb-464c-9d39-b400521639b2.png)

**Comment by slunat** [comment]
> I am not sure if this is related but the preferences button also seems to vanish in this mode.
> 
> ![image](https://user-images.githubusercontent.com/5961364/129741601-7936612b-c0bb-464c-9d39-b400521639b2.png)

Yep I also noticed this. Wasn't sure what caused it but it does line up with me moving the buttons to the right as i could see the preferences fine with it on the left. Also something else I noticed, with the buttons on the left the settings button acts as a hyperlink and opens the account management page in an external browser. With the buttons on the right, the settings button opens account management within the program.
